templates/github/.github/workflows/fips.yml.j2

```diff
@@ -0,0 +1,64 @@
+{% include 'header.j2' %}
+---
+name: Pulp FIPS Nightly CI
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # runs at 4:00 UTC daily
+    - cron: '00 4 * * *'
+
+jobs:
+  vagrant:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test_type: pulp3-source-centos8-fips
+          - test_type: pulp3-source-centos7-fips
+    steps:
+      - uses: actions/checkout@v2.3.1
+        with:
+          submodules: true
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Ansible
+        run: |
+          pip3 install -U pip
+          pip3 install "ansible>2.9,<3"
+          ansible --version
+      - name: Pulp Installer
+        run: |
+          cd ..
+          git clone --depth=1 https://github.com/pulp/pulp_installer
+      - name: Install Vagrant
+        shell: bash
+        run: |
+          cd ../pulp_installer/
+          git submodule update --init
+          {%- if plugin_name in ['pulp_file', 'pulpcore'] %}
+          sed -i "/{{ plugin_name }}/d" .github/workflows/scripts/install.sh
```

Comment:
This looks dangerous to me. Looking at these lines here, it's not even clear to me, what lines are supposed to be deleted. And if that file introduces a new important line containing "pulpcore" it will break in a way that is almost impossible to debug.

Reply:
I agree, it currently deletes git clone of pulpcore and pulp_file, I plan to change the script on the installer side, make it more customizable there

Reply:
Being customizable in the script is great, because then you can see there that there might be an expected interface to it.

Reply:
I've started a PR: https://github.com/pulp/pulp_installer/pull/539